pkg/services/ngalert/api/tooling/definitions/api.go

```diff
@@ -3,23 +3,22 @@
 // Package definitions includes the types required for generating or consuming an OpenAPI
 // spec for the Grafana Alerting API.
 //
+//	 Schemes: http, https
+//	 BasePath: /api/v1
+//	 Version: 1.1.0
 //
-//     Schemes: http, https
-//     BasePath: /api/v1
-//     Version: 1.1.0
+//	 Consumes:
+//	 - application/json
 //
-//     Consumes:
-//     - application/json
+//	 Produces:
+//	 - application/json
 //
-//     Produces:
-//     - application/json
+//	 Security:
+//	 - basic
 //
-//     Security:
-//     - basic
-//
-//    SecurityDefinitions:
-//    basic:
-//      type: basic
+//	SecurityDefinitions:
+//	basic:
+//	  type: basic
```

Comment:
The new formatting rules apparently broke the codegen in ngalert which treats this comment as yaml and does not like tabs
When I run `make` in the tooling dir it fails with `yaml: found character that cannot start any token`

Reply:
That's an upstream issue that was fixed in a later version of the Swagger client, #55159 should make that work again.

It seems to be a bit confused as to the types though, so I haven't added updated Swagger definitions to that PR.